__tests__/app.test.js

```diff
@@ -29,3 +29,16 @@ describe('GET /api/topics', () => {
         });
     });
 });
+
+describe('GET /api/articles/:article_id', () => {
+    test('responds with status: 200 and article object with correct properties', () => {
+        return request(app)
+        .get('/api/articles/5')
+        .expect(200)
+        .then((response) => {
+            const article = response.body;
+            console.log(article)
+            expect(typeof article).toBe('object')
+        })
+    });
```

Comment:
Please add some error handling tests. Think about if someone entered an article id that didn't exist - 5555 [404 error] and if someone entered an invalid path /api/articles/dog [400 error]